openhtf/output/servers/station_server.py

```diff
@@ -0,0 +1,622 @@
+"""HTTP server serving station info and an Angular frontend / web GUI.
+
+This module is comparable to OpenHTF's output.web_gui module. Some differences:
```

Comment:
Probably update the docstring since this is going to replace the old gui.

Reply:
#727 Done